views/components/settings/parts/display-config.es

```diff
@@ -1,13 +1,14 @@
 import path from 'path-extra'
 import fs from 'fs-extra'
-import { shell, remote } from 'electron'
+import { shell } from 'electron'
 import { Grid, Col, Button, ButtonGroup, FormControl, Checkbox, OverlayTrigger, Tooltip } from 'react-bootstrap'
 import { connect } from 'react-redux'
 import React from 'react'
 import PropTypes from 'prop-types'
 import Divider from './divider'
 import { get, debounce } from 'lodash'
 import FontAwesome from 'react-fontawesome'
+import { FolderPickerConfig } from './utils'
```

Comment:
NewWindow Plugin handles this incorrectly, which resulted in a module not found error.